workitem/link/link_repository.go

```diff
@@ -111,6 +113,10 @@ func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targe
 		}
 		return nil, errors.NewInternalError(db.Error.Error())
 	}
+	// save a revision of the created work item link
+	if err := r.revisionRepo.Create(ctx, creatorID, RevisionTypeCreate, *link); err != nil {
+		return nil, errs.WithStack(err)
```

Comment:
@xcoulon The point of wrapping is to add the outside context in which you tried to perform an action. Let's say you want to rename a file from `/tmp/foo/bar` to `/tmp/john/doe`. If the file `/tmp/foo/bar` doesn't exist, the error would just be `file not found /tmp/foo/bar`, whereas if you wrap the error it would have more context to it that helps to see what the actual action was: `failed to rename file /tmp/foo/bar: file not found /tmp/foo/bar`.
Does that help?